lib/standard/string.nit

```diff
@@ -1610,6 +1610,20 @@ class FlatBuffer
 	# Create a new empty string.
 	init do end
 
+	# Create a new buffer with a given nativestring, capcity, and length.
+	#
+	# the size of `items` should be `cap` (or more).
+	# `items` will be then used as is to store the characters of the buffer.
+	#
+	# If `items` is shared, `written` should be set to true after the creation
+	# so that a modification will do a copy-on-write.
```

Comment:
This doc needs reviewing:
* s/capcity/capacity/
* s/the size/The size/
* In the synopsis, you could use the real name of the attributes (and they should have their complete names).